src/structure_managers/make_structure_manager.hh

```diff
@@ -302,39 +302,45 @@ namespace rascal {
   }
 
   namespace internal {
-    template <typename StructureManagerPtr, int TargetLevel>
+    template <typename StructureManager, int TargetLevel>
```

Comment:
Why is it not a pointer anymore? Or is it a pointer and just not named like it any more?

Reply:
std::shared_ptr<StructureManager> is explicitly required